tanjun/__init__.py

```diff
@@ -92,7 +92,6 @@
     "Argument",
     "Option",
     "ShlexParser",
-    "parser_descriptor",
     "verify_parameters",
     "with_argument",
```

Comment:
This is barely in scope but should we have these decorators implicitly initiate a Parser wrapper if they don't find one while following the chain?